2024-04-24 10:21:56,899 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:22:56,997 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:23:57,092 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:24:57,211 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:25:57,322 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:26:57,352 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:27:57,466 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:28:57,555 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:29:57,650 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

